onnx_tf/backend.py

```diff
@@ -254,12 +279,13 @@ def validate_initializer_name(name):
       return name.replace(
           ":", "_tf_") + "_" + get_unique_suffix() if ":" in name else name
 
-    return [(init.name,
-             tf.constant(tensor2list(init),
-                         shape=init.dims,
-                         dtype=data_type.onnx2tf(init.data_type),
-                         name=validate_initializer_name(init.name)))
-            for init in initializer]
+    tensor_dict = [(init.name,
+                    tf.Variable(np.array(tensor2list(init)).reshape(init.dims),
```

Comment:
The initializers should be constant if training_mode is False. For inference, they are not going to change.

Reply:
Yes.  Fixed it.